volunteer/index.html

```diff
@@ -2,12 +2,39 @@
 enable-mentor-alert: false
 title: "Volunteer Application Form"
 scripts: |
-    <script src="js/spring15android.js"></script>
 ---
+<div class="banner-image volunteer">
+    <h1 class="banner-title">Volunteers</h1>
+</div>
+
+<!--Card-->
+<div class="row section volunteer-row">
+  <div class="col-sm-6 col-md-6">
+    <div class="thumbnail volunteer-card">
+      <img src="../images/volunteers/classroomVolunteers.jpg">
+        <div class="caption">
+          <h3>Classroom volunteer</h3>
+          <p>Classroom Volunteers play an essential role by providing one on one support to our students and assisting our instructors. With an understanding of the fundamentals of HTML, CSS, java, and javascript, you will engage a diverse group of students through a project based coding course. </p>
+        </div>
+    </div>
+  </div>
+
+  <div class="col-sm-6 col-md-6">
+    <div class="thumbnail volunteer-card" style="margin-left: 5%;">
+      <img src="../images/volunteers/volunteerPanel.jpg">
+        <div class="caption">
+          <h3>One-Time event volunteers</h3>
+          <p>One-Time Events Volunteer support the Mission Bit staff with activities throughout the year. Events typically occur outside of the classroom, at various locations throughout San Francisco and Oakland (e.g. Demo Day, Mission Bit Gala, Student Admissions Interview, Workshops). </p>
+        </div>
+    </div>
+  </div>
+</div>
+<!--/.Card-->
+<p><a href= "https://docs.google.com/forms/d/e/1FAIpQLSddADoXnDOTw9Y-dlgn47P_hcSyO_BtZpbhB4pntEygKQzMCg/viewform" class="btn btn-lg btn-default shadow hardware volunteer-buttons" role="button">Sign up</a>
 
```

Comment:
Instead of linking to an external page, create a volunteer_form.html which is exactly what the old page used to be: an embedded form in our own pages. Also, delete the old code instead of commenting that out (lines 35 - 57). Github will keep the old versions if we ever need them.